bundles/org.openhab.binding.modbus.helioseasycontrols/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,1188 @@
+		<channels>
+			<!--  Party Mode -->
+			<channel id="partyModeDuration" typeId="partyModeDuration"/>
+			<channel id="partyModeFanStage" typeId="partyModeFanStage"/>
+			<channel id="partyModeRemainingTime" typeId="partyModeRemainingTime"/>
+			<channel id="partyModeStatus" typeId="partyModeStatus"/>
+			<!--  Standby Mode -->
+			<channel id="standbyModeDuration" typeId="standbyModeDuration"/>
+			<channel id="standbyModeFanStage" typeId="standbyModeFanStage"/>
+			<channel id="standbyModeRemainingTime" typeId="standbyModeRemainingTime"/>
+			<channel id="standbyModeStatus" typeId="standbyModeStatus"/>
+			<!--  Holiday Programme -->
+			<channel id="holidayProgramme" typeId="holidayProgramme"/>
+			<channel id="holidayProgrammeFanStage" typeId="holidayProgrammeFanStage"/>
+			<channel id="holidayProgrammeStart" typeId="holidayProgrammeStart"/>
+			<channel id="holidayProgrammeEnd" typeId="holidayProgrammeEnd"/>
+			<channel id="holidayProgrammeInterval" typeId="holidayProgrammeInterval"/>
+			<channel id="holidayProgrammeActivationTime" typeId="holidayProgrammeActivationTime"/>
+			<!--  Operating Mode / Fan Stage-->
+			<channel id="operatingMode" typeId="operatingMode"/>
+			<channel id="fanStage" typeId="fanStage"/>
+			<channel id="percentageFanStage" typeId="percentageFanStage"/>
+			<channel id="extractAirFanStage" typeId="fanStage"/>
+			<channel id="supplyAirFanStage" typeId="fanStage"/>
+		</channels>
+	</channel-group-type>
+	
+	<channel-group-type id="unitConfig">
+		<label>Unit Config</label>
+		<description>Configuration parameters of the ventilation unit</description>
+		<channels>
+			<!--  Fan Stage Configuration -->
+			<channel id="minFanStage" typeId="minFanStage"/>
+			<channel id="assignmentFanStages" typeId="assignmentFanStages"/>
+			<channel id="fanStageStepped0to2v" typeId="fanStageStepped1"/>
+			<channel id="fanStageStepped2to4v" typeId="fanStageStepped2"/>
+			<channel id="fanStageStepped4to6v" typeId="fanStageStepped2"/>
+			<channel id="fanStageStepped6to8v" typeId="fanStageStepped2"/>
+			<channel id="fanStageStepped8to10v" typeId="fanStageStepped2"/>
+			<channel id="voltageFanStage1ExtractAir" typeId="voltageFanStage"/>
+			<channel id="voltageFanStage2ExtractAir" typeId="voltageFanStage"/>
+			<channel id="voltageFanStage3ExtractAir" typeId="voltageFanStage"/>
+			<channel id="voltageFanStage4ExtractAir" typeId="voltageFanStage"/>
+			<channel id="voltageFanStage1SupplyAir" typeId="voltageFanStage"/>
+			<channel id="voltageFanStage2SupplyAir" typeId="voltageFanStage"/>
+			<channel id="voltageFanStage3SupplyAir" typeId="voltageFanStage"/>
+			<channel id="voltageFanStage4SupplyAir" typeId="voltageFanStage"/>
+			<!--  Bypass Configuration -->
+			<channel id="bypassRoomTemperature" typeId="bypassRoomTemperature"/>
+			<channel id="bypassMinOutsideTemperature" typeId="bypassMinOutsideTemperature"/>
+			<channel id="bypassFromDay" typeId="day"/>
+			<channel id="bypassFromMonth" typeId="month"/>

```

Comment:
Can you combine these? Same for below.

Reply:
It's the same beauty as the system date and time... there are 2 different variables in the Modbus protocol spec. But I suppose mapping this to a single DateTime type would improve usability.